app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -84,6 +84,7 @@
           android:layout_gravity="center_vertical"
           android:contentDescription="@{viewModel.computeHintListDropDownIconContentDescription()}"
           android:padding="8dp"
+          android:tint="@color/color_palette_icon_background_color"
```

Comment:
We cannot use `color_palette` color directly in layout-xml files. Please use `component_colors.xml` file colors only here.

Reply:
done.